Sampler/Assets/build.yaml

```diff
@@ -49,6 +50,7 @@ BuildWorkflow:
   test:
```

Comment:
We should add `    - build` to the `pack` task.

Reply:
done